sda-commons-server-testing/src/main/java/org/sdase/commons/server/testing/junit5/EnvironmentExtension.java

```diff
@@ -0,0 +1,75 @@
+package org.sdase.commons.server.testing.junit5;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.sdase.commons.server.testing.Environment;
+
+/**
+ * This Junit 5 extension allows to set environment variables for JUnit tests. All environment
+ * variables will be reset after the test has run.
+ *
+ * <p>Example:
+ *
+ * <pre>
+ *   class MyTest {
+ *
+ *     &#64;RegisterExtension
+ *     public final env = new EnvironmentExtension().setEnv("DISABLE_JWT", "true");
+ *
+ *   }
+ * </pre>
+ */
+public class EnvironmentExtension implements BeforeAllCallback, AfterAllCallback {
```

Comment:
You stated before that we will not provide an extension for setting env variables. Do we still need it?

Reply:
deleted